ui/rating-dialog/src/play/java/de/danoeh/antennapod/ui/ratingdialog/RatingDialogFragment.java

```diff
@@ -0,0 +1,49 @@
+package de.danoeh.antennapod.ui.ratingdialog;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+import com.google.android.material.dialog.MaterialAlertDialogBuilder;
+import de.danoeh.antennapod.ui.ratingdialog.databinding.RatingDialogBinding;
+
+public class RatingDialogFragment extends DialogFragment {
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setCancelable(false);
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
+        return new MaterialAlertDialogBuilder(getContext())
+                .setView(onCreateView(getLayoutInflater(), null, savedInstanceState))
+                .create();
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        RatingDialogBinding viewBinding = RatingDialogBinding.inflate(inflater);
+
+        viewBinding.headerLabel.setText(Html.fromHtml(getString(R.string.rating_tagline, "2020-10-10",
+                "<br/><b><big><big><big><big><big>", 35, "</big></big></big></big></big></b><br/>")));
```

Comment:
`fromHtml` is deprecated here, you'll want to use `HtmlCompat.fromHtml` instead. https://developer.android.com/reference/androidx/core/text/HtmlCompat#fromHtml(java.lang.String,int,android.text.Html.ImageGetter,android.text.Html.TagHandler)